packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -76,15 +76,15 @@
                                     aria-valuemax="100">
                                     <span
                                         class="oui-progress__label float-left"
-                                        data-ng-bind="::convertBytesSize(hosting.quotaUsed.value, hosting.quotaUsed.unit, 2) + ' / ' + convertBytesSize(hosting.quotaSize.value, hosting.quotaSize.unit)"
+                                        data-ng-bind="::convertBytesSize(hosting.quotaUsed.value, hosting.quotaUsed.unit, 2) + ' / ' + convertBytesSize(hosting.quotaSize.value, hosting.quotaSize.unit, 2)"
```

Comment:
It should be a variable instead of doing the same thing two times ^^

Reply:
done :)